src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/appsettings.json

```diff
@@ -8,7 +8,10 @@
     "DelegationsAccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",
     "DelegationsContainer": "delegations",
     "DelegationsBlobEndpoint": "http://127.0.0.1:10000/devstoreaccount1",
-    "BlobLeaseTimeout": 15
+    "BlobLeaseTimeout": 15,
+    "DelegationEventQueueEndpoint": "http://127.0.0.1:10000/devstoreaccount1",
+    "DelegationEventQueueAccountName": "devstoreaccount1",
+    "DelegationEventQueueAccountKey": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
```

Comment:
Disse må vel bestilles deploy/terraform verdier for i keyvault før det kan deployes til noe AT/TT/Prod miljø:
DelegationEventQueueEndpoint
DelegationEventQueueAccountName
DelegationEventQueueAccountKey

Reply:
Avklart med Øystein, men må verifiseres at det henger sammen etter deploy